clients/src/main/java/org/apache/kafka/clients/producer/internals/RecordAccumulator.java

```diff
@@ -401,10 +401,14 @@ public boolean hasUnsent() {
         return batches;
     }
 
+    private Deque<RecordBatch> getDeque(TopicPartition tp) {
```

Comment:
NIT: On a separate thread it was mentioned that we do not follow getter convention. Would it make sense to stick to that here as well?